static/skin/index.js

```diff
@@ -267,35 +279,38 @@
         });
     }
 
+    function noResultsContent(kiwixHomeBody) {
```

Comment:
Function names and signatures must not be misleading. This function name in its current form suggests that it returns some content, whereas it modifies its parameter and some global state too (`loader`). Therefore a better name would be `setNoResultsContent()`. Besides, passing `kiwixHomeBody` as a parameter doesn't make sense - since that creates a wrong impression that all side-effect of that function is limited to the `kiwixHomeBody` element.